content/source/assets/stylesheets/_downloads.scss

```diff
@@ -50,6 +51,38 @@ body.layout-downloads {
           font-size: 40px;
           margin-bottom: -3px;
         }
+
+        .learn-cta {
```

Comment:
Let's have the `a` always have `underline` just so the link is a bit more obvious.